tests/src/test/scala/org/http4s/parser/QueryParserSpec.scala

```diff
@@ -13,6 +11,9 @@ class QueryParserSpec extends Http4sSpec {
     QueryParser.parseQueryString(str)
 
   "The QueryParser" should {
+    import org.specs2.matcher._
+    def be_\/-(q: Query): RightDisjunctionCheckedMatcher[Query] =
```

Comment:
No, unfortunately this is not obvious to me. I would have to play with the code a bit to get an idea.